packages/vue-app/template/utils.js

```diff
@@ -663,3 +663,9 @@ export function stripTrailingSlash (path) {
 export function isSamePath (p1, p2) {
   return stripTrailingSlash(p1) === stripTrailingSlash(p2)
 }
+
+export function setScrollRestoration (newVal) {
```

Comment:
Don't think we need this function, any benifis for not setting `scrollRestoration` as we already have `'scrollRestoration' in window.history` checking ?

Reply:
'scrollRestoration' in window.history` not check for setter of this property.
Angular do this [function](https://github.com/angular/angular/pull/30630/files#diff-bc2334a81b493353fc53a822bec60f8cR255), but i think simple try/catch simpler